modules/exploits/unix/webapp/ajenti_login_rce.rb

```diff
@@ -0,0 +1,77 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => "Ajenti 2.1.31 Remote Code Execution",
```

Comment:
The module name should describe the vulnerability and should *usually* match the module file name (with the caveat that the module file name should not include version numbers).

Also, `Remote` is implied by virtue of the module being an `exploit`.

Also, the vulnerability appears to be a command injection vulnerability, rather than code execution.

For example, this describes the vulnerable software (`Ajanti`), the vulnerable API endpoint (`auth`), the vulnerable parameter (`username`), and the vulnerability type (`Command Execution`) :

```suggestion
                      'Name'            => 'Ajenti 2.1.31 auth username Command Execution',
```
or

```suggestion
                      'Name'            => 'Ajenti auth username Command Execution',
```

or

```suggestion
                      'Name'            => 'Ajenti auth username Unauthenticated Command Execution',
```

The file name should be updated to match the module name (in a shortened format). Something like one of these:

* ajenti_auth_username_command_execution
* ajenti_auth_username_unauth_cmd_exec
* ajenti_auth_username_cmd_exec
* ajenti_auth_username_rce